src/main/java/uk/gov/hmcts/reform/judicialapi/util/RefDataUtil.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.judicialapi.util;
+
+import lombok.Getter;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+
+import static java.util.Objects.isNull;
+
+@Slf4j
+@Getter
+public class RefDataUtil {
+
+    private RefDataUtil() {
+    }
+
+    private static int defaultPageSize;
+
+
+    public static Pageable createPageableObject(Integer page, Integer size) {
+        if (isNull(size)) {
+            size = defaultPageSize;
+        }
+        return PageRequest.of(page, size);
+    }
+
+    @Value("${defaultPageSize}")
+    public static void setDefaultPageSize(int defaultPageSize) {
```

Comment:
Thanks for creating util for creating pageable object. Just one concern. did you check if `defaultPageSize` config value is being set here? I think `@Value` wont be applied for static methods.

I know if you remove static then sonar will complain. It is better to have a parameter `defaultPageSize`  in the method `createPageableObject` so that the caller is responsible to send the defaultPageSize.

Reply:
You can use `@Value` for a static field, you'll just need to provide a setter method; see here: https://stackoverflow.com/a/7253751/12194805

Reply:
yes, we need to provide setter method which is not static, but if we do so then sonar will complain as below
![image](https://user-images.githubusercontent.com/73644938/117295850-3ee79f00-ae6c-11eb-904b-5c28df63e1c9.png)

Reply:
Ahh okay, I wonder why we aren't receiving this issue in [PRD](https://sonarcloud.io/project/issues?id=RDPROFESSIONALAPI&resolved=false) 🤔

Reply:
because of this https://github.com/hmcts/rd-professional-api/blob/master/build.gradle#L262 :P

Reply:
Makes sense 😛

Reply:
I think we can ignore this sonar rule it's perfectly fine in this case.

Reply:
But we have a solution to fix it :) it is really not necessary to have this config variable in this common util class. Let this value come from the caller, so that it will not be tightly coupled if I want to have different default page size for different requirements